webapp/packages/plugin-connections-administration/src/Administration/Connections/ConnectionsAdministrationService.ts

```diff
@@ -75,7 +71,6 @@ export class ConnectionsAdministrationService extends Bootstrap {
       isHidden: () => this.serverConfigResource.distributed,
       getContentComponent: () => ConnectionsAdministration,
       getDrawerComponent: () => ConnectionsDrawerItem,
```

Comment:
why `onActivate` logic was removed?

Reply:
I think its better to load resources in places where there are used. In this case if we delete the component or hook that uses certain resource, the api call also will be deleted. When we load something on onActivate, we cant be sure where certain resources are used